source/common/http/utility.cc

```diff
@@ -329,5 +329,42 @@ const std::string& Utility::getProtocolString(const Protocol protocol) {
   NOT_REACHED;
 }
 
+void Utility::appendToHeader(HeaderString& header, const std::string& data) {
```

Comment:
Use `HeaderMapImpl::appendToHeader`?

Reply:
Sorry,  this is not my change.  I may "rebase" wrong change.  Will fix it.